build.gradle

```diff
@@ -148,8 +146,6 @@ subprojects {
     jar.enabled = true
 
     dependencies {
-        checkstyle "com.puppycrawl.tools:checkstyle:${checkstyleVersion}"
```

Comment:
Why remove this? Otherwise tool version will only be what is packaged with Gradle.

Reply:
Actually the `checkstyle.toolVersion = "${checkstyleVersion}"` does the trick. Just tested with older version of Checkstyle.